cloudformation/template.yml

```diff
@@ -0,0 +1,339 @@
+Description: A CloudFormation template for deploying Raster Vision Batch jobs to AWS.
+
+Metadata:
+  'AWS::CloudFormation::Interface':
+    ParameterGroups:
+      -
+        Label:
+          default: Project Configuration
+        Parameters:
+          - Project
+          - Environment
+          - Region
+      -
+        Label:
+          default: EC2 Instance Configuration
+        Parameters:
+          - VPC
+          - SubnetIds
+          - KeyName
+          - AMI
+          - InstanceTypes
+          - MinimumVCPUs
+          - DesiredVCPUs
+          - MaximumVCPUs
+      -
+        Label:
+          default: Docker Container Configuration
+        Parameters:
+          - RepositoryName
+          - ImageName
+          - ImageTag
+          - InstanceVCPUs
+          - InstanceMemory
+      -
+        Label:
+          default: Batch Configuration
+        Parameters:
+          - JobDefinitionName
+          - BatchServiceRolePolicyARN
+          - SpotFleetServiceRolePolicyARN
+    ParameterLabels:
+      Project:
+        default: Name
+      Environment:
+        default: Environment
+      Region:
+        default: AWS Region
+      AMI:
+        default: AMI
+      KeyName:
+        default: SSH Key Name
+      SpotFleetBidPercentage:
+        default: Spot Fleet Bid Percentage
+      MinimumVCPUs:
+        default: Minimum vCPU Count
+      DesiredVCPUs:
+        default: Desired vCPU Count
+      MaximumVCPUs:
+        default: Maximum vCPU Count
+      InstanceTypes:
+        default: Instance Types
+      JobDefinitionName:
+        default: Job Definition Name
+      InstanceVCPUs:
+        default: vCPU Limit
+      InstanceMemory:
+        default: Memory Limit
+      BatchServiceRolePolicyARN:
+        default: Batch Service Role
+      SpotFleetServiceRolePolicyARN:
+        default: Spot Fleet Service Role
+      RepositoryName:
+        default: Repository Name
+      ImageName:
+        default: Image Name
+      ImageTag:
+        default: Image Tag
+      VPC:
+        default: VPC
+      SubnetIds:
+        default: Subnets
+
+Parameters:
+  Project:
+    Type: String
+    Default: Raster Vision CloudFormation
+    Description: Name of the project associated with the Batch job
+
+  Environment:
+    Type: String
+    Default: Testing
+    Description: Environment of the project being deployed (e.g. Staging, Development, Production)
+
+  Region:
+    Type: String
+    Default: 'us-east-1'
+    Description: AWS Region in which to deploy the Batch job
+
+  AMI:
+    Type: String
+    Default: 'ami-06681f03ed7f21643'
+    Description: Amazon Machine Image to use for the compute environment
+
+  KeyName:
+    Type: 'AWS::EC2::KeyPair::KeyName'
```

Comment:
I think that we can safely remove the quoting here.

Reply:
Roger! I went for it because I was worried that `:` always acted as a special character in YAML, but [that appears not to be true](https://stackoverflow.com/a/33149072).